fdmi/ut/sd_common.c

```diff
@@ -81,8 +78,8 @@ void fdmi_serv_start_ut(const struct m0_filterc_ops *filterc_ops)
 	sd_ut_lfile = fopen(LOG_FILE_NAME, "w+");
 	M0_UT_ASSERT(sd_ut_lfile != NULL);
 
-	rc = m0_cs_init(&g_sd_ut.motr, sd_ut_xprts, ARRAY_SIZE(sd_ut_xprts),
-			sd_ut_lfile, false);
+	rc = m0_cs_init(&g_sd_ut.motr, m0_net_all_xprt_get(), 
```

Comment:
Please remove the trailing space at the end of this line.

Reply:
incorporated